packages/nest/src/schematics/nestjs-schematics/nestjs-schematics.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  chain,
+  externalSchematic,
+  Rule,
+  Tree,
+} from '@angular-devkit/schematics';
+import { getProjectConfig, toFileName } from '@nrwl/workspace';
+import { NestSchematicsSchema } from './schema';
+
+interface NormalizedSchema extends NestSchematicsSchema {
+  name: string;
+  sourceRoot: string;
+}
+
+function normalizeOptions(
+  host: Tree,
+  options: NestSchematicsSchema
+): NormalizedSchema {
+  if (options.directory && !options.path) {
+    options.path = options.directory;
+  }
+
+  const { sourceRoot } = getProjectConfig(host, options.project);
+  const name = toFileName(options.name);
+
+  return {
+    ...options,
+    name,
+    path: options.path,
+    sourceRoot,
+    spec: options.spec ? options.spec : options.unitTestRunner === 'jest',
```

Comment:
to be in line with how nest is expecting the cli params would we rather want to do this?

```
...(options.spec && options.unitTestRunner === 'jest' ? {spec: true} : {'no-spec': true}
```

Taking a look at the docs there is an option for `--spec` and `--no-spec` where the former is the default
https://docs.nestjs.com/cli/usages#nest-generate

Not sure if it really makes much difference but piping in using the nest specific cli params might not be the worst idea.

Reply:
Oh, that's weird then. I went though all the schemas that are in the nest schematics, and I don't remember seeing the `no-spec` flag. I'll double check.

Reply:
Hmm that is odd. Not sure why the schematics and the docs would differ there. I didn't check the actual schematic code. If it doesn't exist there then this is a moot point.